apps/wallet/src/ui/app/components/accounts/ImportRecoveryPhraseForm.tsx

```diff
@@ -125,14 +125,15 @@ export function ImportRecoveryPhraseForm({
             </div>
 
             <div className="sticky bottom-0 left-0 flex flex-col gap-2.5 bg-neutral-100 pt-sm dark:bg-neutral-6">
-                {touchedFields.recoveryPhrase && errors.recoveryPhrase && (
-                    <InfoBox
-                        type={InfoBoxType.Default}
-                        supportingText={errors.recoveryPhrase.message}
-                        icon={<Exclamation />}
-                        style={InfoBoxStyle.Elevated}
-                    />
-                )}
+                {errors?.recoveryPhrase?.message &&
+                    !touchedFields.recoveryPhrase?.every(Boolean) && (
+                        <InfoBox
+                            type={InfoBoxType.Error}
+                            supportingText={errors?.recoveryPhrase?.message}
```

Comment:
Lets update https://github.com/iotaledger/iota/blob/921db2eb6e52b9c26bba5b4f0453a3b9683ec0f3/apps/wallet/src/ui/app/components/accounts/ImportRecoveryPhraseForm.tsx#L31 to `Mnenonic is invalid` so the message is in sync with what how the app calls it